### ISSUE TITLE
Add data from local computer

### ISSUE THREAD

**Issue by Clement-Lef** [SwissDataScienceCenter/renku-python#453]
## Describe the bug
Trying to add a dataset that is stored locally on my computer. I tried to add it to the git repo using LFS then used renku commands to add it to a renku dataset but it doesn't seems to work this way. The project is https://renkulab.io/projects/382/.

## Details

**Renku version:** 0.3.4.dev50
**OS:** Linux (#1 SMP Mon Oct 22 10:40:32 EDT 2018)
**Python:** 3.6.7

### Traceback

```
Traceback (most recent call last):
  File "[...]/site-packages/renku/cli/_exc.py", line 41, in main
    return super().main(*args, **kwargs)
  File "[...]/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "[...]/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "[...]/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "[...]/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "[...]/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "[...]/site-packages/renku/cli/_client.py", line 75, in new_func
    result = ctx.invoke(method, client, *args, **kwargs)
  File "[...]/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "[...]/site-packages/renku/cli/dataset.py", line 150, in add
    force=force,
  File "[...]/site-packages/renku/api/datasets.py", line 128, in add_data_to_dataset
    dataset, dataset_path, url, target, **kwargs
  File "[...]/site-packages/renku/api/datasets.py", line 291, in _add_from_git
    submodule_path.relative_to(self.path).as_posix()
  File "[...]/site-packages/git/cmd.py", line 548, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "[...]/site-packages/git/cmd.py", line 1014, in _call_process
    return self.execute(call, **exec_kwargs)
  File "[...]/site-packages/git/cmd.py", line 825, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git submodule add --force --name test-upload-local2 /home/jovyan/test-upload-local2 .renku/vendors/local/home/jovyan/test-upload-local2
  stderr: 'Cloning into '/home/jovyan/test-upload-local2/.renku/vendors/local/home/jovyan/test-upload-local2'...
done.
Downloading iris.data.txt (4.7 KB)
Error downloading object: iris.data.txt (6d75fc9): Smudge error: Error downloading iris.data.txt (6d75fc95493561b905a31ba73425a2858a2d23e6cd64b145b10d18008b980a2e): batch request: missing protocol: "/home/jovyan/test-upload-local2.git/info/lfs"

Errors logged to /home/jovyan/test-upload-local2/.git/modules/test-upload-local2/lfs/logs/20190308T153719.969878057.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: iris.data.txt: smudge filter lfs failed
Unable to checkout submodule '.renku/vendors/local/home/jovyan/test-upload-local2''
```

## Additional context
Here is a screen of the commands that I used:

<img width="558" alt="uploadlocalcommands" src="https://user-images.githubusercontent.com/34097650/54039060-48104f00-41c2-11e9-96d3-29e4e833b77f.PNG">

**Comment by jirikuncar** [comment]
@Clement-Lef please try with latest renku-python: https://renkulab.io/gitlab/clement.lefebvre/test-upload-local2/merge_requests/1

**Comment by jirikuncar** [comment]
fixed in #442
